etc/omero.properties

```diff
@@ -638,6 +638,38 @@ omero.ldap.new_user_group=default
 ## omero.ldap.new_user_group=:bean:myNewUserGroupMapperBean
 
 #############################################
+## OMERO client properties:
+#############################################
+
+## Serverside scripts used in IScript service Clients shouldn't display.
```

Comment:
The double `##` means the property description will not be parsed to auto-generate the documentation page. Can we revert to `#` instead?